src/components/v2/VoteProposalUi/styles.ts

```diff
@@ -0,0 +1,146 @@
+import { css } from '@emotion/react';
+import { alpha, useTheme } from '@mui/material';
+
+export const useStyles = () => {
+  const theme = useTheme();
+
+  return {
+    root: css`
+      padding-top: 0;
+      padding-bottom: 0;
+      ${theme.breakpoints.down('sm')} {
+        padding-left: 0;
+        padding-right: 0;
+      }
+    `,
+    gridItem: css`
+      padding: ${theme.spacing(6, 0)};
+      ${theme.breakpoints.down('sm')} {
+        padding-left: ${theme.spacing(6)};
+        padding-right: ${theme.spacing(6)};
+      }
+    `,
+    gridItemLeft: css`
+      padding-right: ${theme.spacing(6)};
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+    `,
+    cardHeader: css`
+      display: flex;
+      justify-content: space-between;
+    `,
+    cardBadges: css`
+      /* TODO */
+    `,
+    cardBadgeItem: css`
+      padding: ${theme.spacing(1, 3)};
+      background-color: ${theme.palette.secondary.light};
+      border-radius: ${theme.shape.borderRadius.small}px;
+      margin-right: ${theme.spacing(2)};
+    `,
+    cardBadgeNumber: css`
+      /* TODO */
+    `,
+    cardBadgeActive: css`
+      background-color: ${alpha(theme.palette.interactive.success as string, 0.1)};
+      color: ${theme.palette.interactive.success};
+    `,
+    cardTitle: css`
+      margin-top: ${theme.spacing(5)};
+      margin-bottom: ${theme.spacing(6)};
+      overflow: hidden;
+      text-overflow: ellipsis;
+      display: -webkit-box;
+      -webkit-line-clamp: 2;
```

Comment:
Vendor prefixes should get added in post css processing

Reply:
but it wasn't. so I had to add it manually here

Reply:
here is an opened issue:
https://github.com/postcss/autoprefixer/issues/1322